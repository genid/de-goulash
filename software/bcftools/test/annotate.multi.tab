1	752566	G	*	.
1	752566	G	T	GT
1	752566	G	C	GC
1	752566	G	A	GA
