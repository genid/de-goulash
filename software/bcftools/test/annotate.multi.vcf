##fileformat=VCFv4.2
##INFO=<ID=ANN,Number=.,Type=String,Description="Some annotation">
##contig=<ID=1>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	752566	.	G	A,C,T	.	.	.
1	752566	.	G	A,C	.	.	.
