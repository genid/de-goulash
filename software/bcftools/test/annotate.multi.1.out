##fileformat=VCFv4.2
##FILTER=<ID=PASS,Description="All filters passed">
##INFO=<ID=ANN,Number=.,Type=String,Description="Some annotation">
##contig=<ID=1>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
1	752566	.	G	A,C,T	.	.	ANN=GT,GC,GA
1	752566	.	G	A,C	.	.	ANN=GC,GA
